src/include/miopen/utility/transposing_solver.hpp

```diff
@@ -77,7 +77,12 @@ struct TransposeInvokeParams : InvokeParams
 
 struct TransposePseudoSolver
 {
-    virtual ~TransposePseudoSolver()                                        = default;
+    TransposePseudoSolver() {}
+    virtual ~TransposePseudoSolver()                    = default;
+    TransposePseudoSolver(const TransposePseudoSolver&) = default;
+    TransposePseudoSolver(TransposePseudoSolver&&)      = default;
+    TransposePseudoSolver& operator=(const TransposePseudoSolver&) = default;
+    TransposePseudoSolver& operator=(TransposePseudoSolver&&)               = default;
     virtual std::string GetTranspose() const                                = 0;
     virtual ConvSolution GetSolution(const ExecutionContext& ctx,
                                      const TransposeProblem& problem) const = 0;
```

Comment:
```suggestion
    virtual ~TransposePseudoSolver()                    = default;

    virtual std::string GetTranspose() const                                = 0;
    virtual ConvSolution GetSolution(const ExecutionContext& ctx,
                                     const TransposeProblem& problem) const = 0;
protected:
    TransposePseudoSolver() = default;
    TransposePseudoSolver(const TransposePseudoSolver&) = default;
```

Reply:
It is enough to add protected default constructor and protected copy constructor